### PR TITLE
Create BaseYii stub

### PR DESCRIPTION
**Pull request by WinterSilence**

### CODE REVIEW

File: stubs/BaseYii.phpstub [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
```
Comment (AdeAttwood) [inline stubs/BaseYii.phpstub:1]:
```suggestion
<?php
/**
 * Copyright 2021 Practically.io All rights reserved
 *
 * Use of this source is governed by a BSD-style
 * licence that can be found in the LICENCE file or at
 * https://www.practically.io/copyright/
 */
declare(strict_types=1);
```

Can we get the file header in here, please?

https://github.com/Practically/PsalmPluginYii2/blob/1.x/stubs/ActiveQuery.phpstub#L1-L9

**Review by AdeAttwood — CHANGES_REQUESTED** [review]
This is cool, thanks for the contribution.

If we can get some tests on this, that will be perfect. Maybe a test with it failing when you pass an integer to the function?

I don't have the actions set up on PRs, so I will get that fixed, then if we can get this rebased, so we can get all the tests checked. The only thing I can see an issue with is the commit message. If you fancy it you can sort that out, if not I will sort that when merging.

**Comment by WinterSilence** [comment]
@AdeAttwood I take that code from package https://github.com/proget-hq/phpstan-yii2

**Comment by AdeAttwood** [comment]
This code is good as is, just needs some tests. @WinterSilence would you like me to finish this PR off and push the changes to `WinterSilence/PsalmPluginYii2`

**Comment by WinterSilence** [comment]
@AdeAttwood  
> would you like me to finish this PR off and push the changes to WinterSilence/PsalmPluginYii2

don't understand you

**Comment by WinterSilence** [comment]
@AdeAttwood don't know how test stubs, but if is work fine in phpstan plugin it's must work fine in psalm plugin too

**Comment by AdeAttwood** [comment]
> but if is work fine in phpstan plugin it's must work fine in psalm plugin too

Unfortunately to often this is not the case. I will get some test done and get this merged in the next couple of days.

**Comment by WinterSilence** [comment]
@AdeAttwood merged with stub generated by `psalm --generate-stubs`

**Comment by AdeAttwood** [comment]
@WinterSilence thanks for this, sorry for the delay it has been put in with #9 with the tests. If you want to see how the tests are done for the stubs, then you can see them [here](https://github.com/Practically/PsalmPluginYii2/pull/9/commits/21c491e2f5f829c6aba210b6d3c9d83b508207a3)

**Pull request closed without merging**